src/processors/BaseProcessor.h

```diff
@@ -221,6 +295,8 @@ class BaseProcessor : private JuceProcWrapper
 
     juce::Array<int> inputModulationPorts {};
     juce::Array<int> outputModulationPorts {};
```

Comment:
Eventually we should remove these, but maybe it's better to do that in the next PR?

Reply:
Yep, for sure, was just leaving them for now whilst I work out what else needs to change to accommodate the portType vectors in the editor etc